src/index.js

```diff
@@ -70,6 +70,10 @@ export default async function NuxtPressModule (options) {
     next()
   })
 
+  this.extendBuild((config, { isClient }) => {
+
```

Comment:
@pimlie this can be removed right?

Reply:
yeah, sry. Must be a leftover from testing something